### ISSUE TITLE
Questions about loss function modification

### ISSUE THREAD

**Issue by nywang2019** [NVIDIA/partialconv#20]
Hi, i want to use and modify your code for image inpainting. I have a dataset with thousands of pics.  Here are some questions,hope someone can help  me: @liuguilin1225 
1.if I want to fill a hole in image A with  more content from image A and less imaginary content from other images. What should i do? Is it right to modify total loss function? Is so, which part of loss should be modified?
2. If I modify loss function, does it work if I train my own dataset both with the pretrained model this repo provided and the modified loss function?
